### PR TITLE
feat : Improvements for SketchApplication.java

### PR DESCRIPTION
**Pull request by Yamenher**
Improvements for better Unhandled errors catching on newer android versions as `AlarmManager` Component is not responsive on recent android versions, I think this code belongs to @JavkhlanK 
 originally :/.

### CODE REVIEW

**Comment by remaker17** [comment]
Duplicate of #1148.

**Comment by Yamenher** [comment]
No, why did you just close it? this modification is made to the app itself, not for the projects class it generates, as this detection is for error detection while using Sketchware and not the apps it makes

**Comment by remaker17** [comment]
> No, why did you just close it? this modification is made to the app itself, not for the projects class it generates, as this detection is for error detection while using Sketchware and not the apps it makes

Sorry for that, I'll reopen this RP then, okay?

File: SketchwareMaterialRedesign [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (remaker17) [inline SketchwareMaterialRedesign:1]: What is that file? Seems unwanted file. Can you remove it?

**Review by remaker17 — CHANGES_REQUESTED** [review]

File: SketchwareMaterialRedesign (hunk 1 above)
Comment (Yamenher) [reply SketchwareMaterialRedesign:1]: idk, I didn't make such a file

**Review by Yamenher — COMMENTED** [review]
:/

File: SketchwareMaterialRedesign (hunk 1 above)
Comment (Yamenher) [reply SketchwareMaterialRedesign:1]: it was made automatically

**Review by Yamenher — COMMENTED** [review]
:/

**Review by Yamenher — COMMENTED** [review]
Alright now, I fixed everything :)

File: SketchwareMaterialRedesign (hunk 1 above)
Comment (remaker17) [inline SketchwareMaterialRedesign:1]: This file is still not deleted

**Review by remaker17 — CHANGES_REQUESTED** [review]

**Comment by remaker17** [comment]
Why did you close this PR? I'm not going to review again.

**Pull request closed without merging**